src/stars.css

```
@@ -0,0 +1,32 @@
+.product__rating {
+  display: flex;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  list-style: none;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:not(:last-child) {
```

Interesting solution, but maybe we can do them by the same pattern?
```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
............. the same ) {
```

---

done